src/components/PageNavigationButtons/index.jsx

```diff
@@ -1,24 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { Button } from '@edx/paragon';
+import { Icon } from '@edx/paragon';
 
 import './PageNavigationButtons.scss';
 
+const PrevButtonText = () => (
+  <div className="btn-label">
+    <Icon className="fa fa-chevron-left" hidden />
+    <span className="btn-text"> Previous </span>
+  </div>
+);
+
+const NextButtonText = () => (
+  <div className="btn-label">
+    <span className="btn-text">Next</span>
+    <Icon className="fa fa-chevron-right" hidden />
+  </div>
+);
 
 const PageNavigationButtons = props => (
   <div className="page-nav-btns">
     {
       props.prev.trim() ?
-        <Link className="btn btn-primary" to={props.prev}>Previous</Link>
+        <Link className="btn btn-outline-secondary" to={props.prev}>
+          {PrevButtonText()}
+        </Link>
       :
-        <Button buttonType="disabled" label="Previous" />
+        <a href="." className="btn btn-outline-secondary disabled" aria-disabled="true">
```

Comment:
Why a period? I'm wondering if we can just do an empty span instead of an anchor tag. Right now it's still clickable.

Reply:
ah yeah i wasn't sure what the right thing to do was here, i wanted to signal to screen readers that this was a disabled link and i wasn't sure if an empty span would do the same thing